### PR TITLE
Huda

### PR DESCRIPTION
**Pull request by kpslp**

### CODE REVIEW

File: docs/whats-new/staging-release-notes.md [hunk 1]
```diff
@@ -2,14 +2,70 @@
 
 > **Tip**: Click the triangle to view more details about the change.
 
+## Staging release 15 March 2023
+
+Frontend version: 1.0.0-20230314.0632 | Backend version: 1.54.0-230309.0902
```
Comment (JLeeChan) [inline docs/whats-new/staging-release-notes.md]: is this the correct version? in DEV the backend version is 1.54.1-230312.1328

File: docs/log-in-as-po.md [hunk 2]
```diff
@@ -1,24 +1,55 @@
...
+1. Go to the [Docs portal](https://docs.developer.tech.gov.sg/) and click **Log in**.
 
-If you are signing in from your GMD, you will be prompted to authenticate your WOG account.
+<kbd>![new-mfa](assets/images/onboarding/po-non-se/new-docs-portal.png)</kbd>
 
-3. Enter the verification code displayed for your SG Govt M365 profile on your Authenticator app.
+2. Click **Log in with TechPass**.
 
-<kbd>![verify-wog-for-po](assets/images/access-sgts-services-using-techpass/verification-code-po.png)-->
+<kbd>![new-mfa](assets/images/onboarding/po-non-se/new-docs.png)</kbd>
```
Comment (JLeeChan) [inline docs/log-in-as-po.md]: image

File: docs/log-in-as-po.md [hunk 3]
```diff
@@ -1,24 +1,55 @@
...
+4. You will be prompted to authenticate your WOG account. A number will be shown on your browser.
 
-4. On the Authenticator app, enter the number shown, and tap **Yes** to authenticate this sign-in.
+<kbd>![new-mfa](assets/images/onboarding/po-non-se/new-mfa.png)</kbd>
 
 You are now successfully logged in with TechPass.
 
 <kbd>![after-po-login](assets/images/access-sgts-services-using-techpass/final.png)</kbd>
 
 ?> To log out of TechPass, click **Log Out** and confirm which account you would like to log out.
```
Comment (JLeeChan) [inline docs/log-in-as-po.md:27]: is the ?>  interntional ?
image

File: docs/log-in-as-po.md [hunk 4]
```diff
@@ -1,24 +1,55 @@
...
+
+**To log in with a GMD device**
+
+1. Go to the [Docs portal](https://docs.developer.tech.gov.sg/) and click **Log in**.
+
+<kbd>![new-mfa](assets/images/onboarding/po-non-se/new-docs-portal.png)</kbd>
+
+2. Click **Log in with TechPass**.
+
+<kbd>![new-mfa](assets/images/onboarding/po-non-se/new-docs.png)</kbd>
```
Comment (JLeeChan) [inline docs/log-in-as-po.md:37]: image

File: docs/log-in-as-po.md [hunk 5]
```diff
@@ -1,24 +1,55 @@
...
+
+3. Sign in to your TechPass account or choose your TechPass account.
+
+<kbd>![sign-in](assets/images/access-sgts-services-using-techpass/log-in-with-techpass.png)</kbd>
+
+4. You will be prompted to authenticate your WOG account. Enter the verification code displayed for your SG Govt M365 profile on your Authenticator app.
```
Comment (JLeeChan) [inline docs/log-in-as-po.md:43]: there's 2 4s and the image is not showing up
image

File: docs/log-in-as-po.md [hunk 6]
```diff
@@ -1,24 +1,55 @@
...
+
+4. You will now be prompted to approve your WOG sign-in. A number will be shown on your browser.
+
+<kbd>![new-mfa](assets/images/onboarding/po-non-se/new-mfa.png)</kbd>
+
+You are now successfully logged in with TechPass.
+
+<kbd>![after-po-login](assets/images/access-sgts-services-using-techpass/final.png)</kbd>
+
+?> To log out of TechPass, click **Log Out** and confirm which account you would like to log out.
```
Comment (JLeeChan) [inline docs/log-in-as-po.md:55]: image

File: docs/whats-new/staging-release-notes.md (hunk 1 above)
Comment (kpslp) [reply docs/whats-new/staging-release-notes.md]: @JLeeChan yinqi mentioned this:
 apologies, there's update to FE version, latest is FE 1.0.0-20230314.0632 BE 1.54.0-230309.0902

File: docs/log-in-as-po.md (hunk 2 above)
Comment (kpslp) [reply docs/log-in-as-po.md]: sorry, it appeared fine on local preview, will fix this, same goes for the rest of text.

Comment (kpslp) [reply docs/log-in-as-po.md]: i mean, images.

File: docs/log-in-as-po.md (hunk 3 above)
Comment (kpslp) [reply docs/log-in-as-po.md:27]: it is supposed to look like this. ?> is supposed to represent information symbol.
 
 @JLeeChan

File: docs/log-in-as-po.md (hunk 6 above)
Comment (kpslp) [reply docs/log-in-as-po.md:55]: likewise as above @JLeeChan

File: docs/log-in-as-po.md (hunk 5 above)
Comment (kpslp) [reply docs/log-in-as-po.md:43]: fixed. as mentioned, images worked fine on local preview. will fix this.

File: docs/log-in-as-po.md (hunk 3 above)
Comment (JLeeChan) [reply docs/log-in-as-po.md:27]: i see, so the View file functionality not working correctly

File: docs/log-in-as-po.md [hunk 7]
```diff
@@ -1,24 +1,55 @@
 # Log in as public officer
 
-1. Go to the [Docs portal](https://docs.developer.tech.gov.sg/) and click **Log in with TechPass**.
-<kbd>![log-in-with-techpass](assets/images/access-sgts-services-using-techpass/first.png)</kbd>
-2. Sign in to your TechPass account or choose your TechPass account.
-<kbd>![sign-in](assets/images/access-sgts-services-using-techpass/log-in-with-techpass.png)</kbd>
+Public officer can log in to TechPass using your non-SE GSIB device or your Government Managed Device (GMD).
```
Comment (SenthilKumarSivakami) [inline docs/log-in-as-po.md:3]: @kpslp to make it clear, can we say  PO can log in with TechPass ...as log in to TechPass may also mean TechPass portal and currently cant do that using GMD.

Comment (SenthilKumarSivakami) [reply docs/log-in-as-po.md:3]: instead of 
Public officer - change to - Public officers

 "your " to "their"

File: docs/log-in-as-po.md [hunk 8]
```diff
@@ -1,24 +1,55 @@
...
-<kbd>![verify-wog-for-po](assets/images/access-sgts-services-using-techpass/verification-code-po.png)-->
+<kbd>![new-mfa](assets/images/onboarding/po-non-se/new-docs-tech.png)</kbd>
 
-You will now be prompted to approve your TechPass sign-in. A number will be shown on your browser.
+3. Sign in to your TechPass account or choose your TechPass account.
 
- <kbd>![number-mfa](assets/images/onboarding/po-non-se/mfa-number-displayed-on-screen.png)</kbd>
+<kbd>![sign-in](assets/images/access-sgts-services-using-techpass/log-in-with-techpass.png)</kbd>
+
+4. You will be prompted to authenticate your WOG account. A number will be shown on your browser.
```
Comment (SenthilKumarSivakami) [inline docs/log-in-as-po.md:19]: @kpslp there are three use cases:

you need to capture the exp of the user for all the three in this step.

Comment (SenthilKumarSivakami) [reply docs/log-in-as-po.md:19]: Use case 1:

1. Log in to GSIB.
2. try to access a resource behind TechPass.
3. If i am not logged into WOG account or session timed out, i will be prompted to enter pwd (GSIB pwd)
4. pwd success, then i need to authenticate by keying the number i see right?
5. no need to authenticate TP profile.
6. access the resource now.

Use case 2:

1. Log in to GSIB.
2. try to access a resource behind TechPass.
3. If i am already logged into WOG account (session still active)
4. Enter the authenticate  number  you see.
5. no need to authenticate TP profile.
6. access the resource now.


use case 3:

1. Log in to GMD.
2. try to access a resource behind TechPass.
3. enter WoG verification code (6 digit OTP)  and then enter the number  displayed for TP .
5. access the resource now.

**Review by SenthilKumarSivakami — CHANGES_REQUESTED** [review]

**Review by JLeeChan — APPROVED** [review]

**Pull request closed without merging**